BAZProjectC1/Models/Models.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Models.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 15/09/22.
+//
+
+import Foundation
+
+public struct MovieApiResponseModel: Codable {
+    let page: Int
+    let results: [MovieModel]
+    let total_pages: Int
+    let total_results: Int
+}
+public struct MovieModel: Codable {
```

Comment:
Deja un espacio entre las llaves y la definición de tu siguiente struct.
```suggestion
}

public struct MovieModel: Codable {
```